### ISSUE TITLE
Adding averaged loss or current loss on tzip/tqdm. 

### ISSUE THREAD

**Issue by TParcollet** [speechbrain/speechbrain#123]
I think it is important to also report a running averaged loss or current mini-batch loss on the progress bar. For instance, I'm dealing with 1000+ hours of training data, one epoch takes more than 4 hours. I definitely don't want to wait 4H to debug the fact that the loss went to NaN :P What do you think?
